plugins/source/facebookmarketing/README.md

```diff
@@ -0,0 +1,11 @@
+# Facebook Marketing Source Plugin
+
+The Facebook Marketing source plugin extracts information from the [Facebook Marketing API](https://developers.facebook.com/docs/marketing-api/reference/v16.0).
+
+## Links
+
+- [User Guide](https://cloudquery.io/docs/plugins/sources/hubspot/overview)
+
+## Development
+
+Much of the code here was "auto-generated" with https://gist.github.com/shimonp21/a4a79c744d045cd95a4569b2fba2013f. It requires cloning https://github.com/facebook/facebook-business-sdk-codegen as a prerequisite.
```

Comment:
```suggestion
Most of the code here was auto-generated with https://gist.github.com/shimonp21/a4a79c744d045cd95a4569b2fba2013f. It requires cloning https://github.com/facebook/facebook-business-sdk-codegen as a prerequisite.
```

Reply:
Maybe also add step by step docs on how to generate the code?

Reply:
I would argue we can just drop this section as it's not something we can maintain re-generate once we push a first version, so users will need to manually edit this anyway.

Reply:
That works too 🚀

Reply:
I made the section shorter, but I think the script may be useful to future contributors (at least so they know where to find facebook's computer-readable API spec.